src/Illuminate/Auth/Passwords/PasswordBrokerManager.php

```diff
@@ -95,7 +95,8 @@ protected function createTokenRepository(array $config)
             $this->app['hash'],
             $config['table'],
             $key,
-            $config['expire']
+            $config['expire'],
+            $config['timeout']
```

Comment:
Definitely a breaking change.

Since current configs doesn't contain this settings this would break after an update with array offset error. Consider providing a default for it if the target is 6.x.

Also timeout doesn't sounds good for me maybe retry limit or something similar.

This should be included in the `laravel/laravel` repo if this gets merged.

Reply:
Maybe the best solution would be to provide null here and check the recently created entries if it's not null.

This would make it disabled by default for all existing installations and enabled by default for all new ones, this would solve the missing translation problem too, because if someone enables them he/she can also add the translation.